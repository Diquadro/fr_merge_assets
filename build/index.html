<!DOCTYPE html><html lang="en"><head><title>My Site</title><link href="css/p4.067b926a.css" rel="stylesheet">
<script src="js/base.d89dccf4.js"></script><link rel="stylesheet" href="css/base.e5548910.css"></head><body><h1>Hello World!</h1><div>P1</div><script src="js/p1.6a25b948.js"></script><link rel="stylesheet" href="css/p1.8bbfdf28.css"><div>P4</div><script src="js/p4.bd479460.js"></script><div>P2</div><script src="js/p2.d6519ec7.js"></script><link rel="stylesheet" href="css/p2.8292d241.css"><div>P4</div><script src="js/p4.bd479460.js"></script><div>P3</div><script src="js/p3.c8376e21.js"></script><link rel="stylesheet" href="css/p3.4e12d1fa.css"><script src="js/index.fe495907.js"></script><link rel="stylesheet" href="css/index.8f2d284e.css"></body></html>