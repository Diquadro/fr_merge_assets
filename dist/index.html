<!DOCTYPE html><html lang="en"></html><head><title>Hello World</title><link href="img/favicon.edda23bf.ico" rel="shortcut icon"><link href="css/styles.81358e82.css" rel="stylesheet"><script src="js/main.5317c1f6.js" defer="defer"></script></head><body><h1>Hello Pug World!</h1><img src="img/picture.7b396424.png"><p>It's generated using <a href="https://github.com/webdiscus/html-bundler-webpack-plugin">HTML Builder Plugin</a> for Webpack.</p></p><div>PARTIAL_1</div></body>