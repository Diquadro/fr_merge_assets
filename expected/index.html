<!DOCTYPE html>
<html lang="en">
  <head>
    <title>My Site</title>
    <link href="css/index.bundle.########.css" rel="stylesheet" />
    <script src="js/index.bundle.########.js" defer></script>
  </head>
  <body>
    <h1>Hello World!</h1>
    <div>P1</div>
    <div>P4</div>
    <div>P2</div>
    <div>P4</div>
    <div>P3</div>
  </body>
</html>
